how_to_build.md

```diff
@@ -100,10 +100,10 @@ available:
 
 * `full`: the default profile, builds all supported target frameworks supported
   on the current platform
-* `net462`: builds only the .NET Framework 4.6.2 target framework
+* `net462`: builds and tests only the .NET Framework 4.6.2 target framework

```

Comment:
Just thought of something. Maybe we should rename this profile to `netfx` or `netframework` or something similar. This way, if we change the .NET Framework version again, we won't have to rename it.

Reply:
True. Let's leave it as is, then.